config/locales/en.yml

```diff
@@ -436,7 +436,8 @@
       schools:
         who_to_contact:
           create:
-            success: "Saved. We will email %{email_address} shortly"
+            failure: "Saved. We will email %{email_address} shortly"
```

Comment:
What are the failure cases here?

Reply:
It's this case:

https://github.com/DFE-Digital/get-help-with-tech/pull/500/commits/1d28c9a16cc75f83429ff1257c04c96d7b1f2a96

when the contact about to be invited is already a user.

Reply:
I don't think I'm really clear about the intended behaviour here - if I've followed this correctly, then when the school invites another person who is _not_ already on the system, it should:

1. Add a User record for them 
2. Send them an invite email
3. Update the preorder information status to "School contacted"

But if the person _is_ already on the system, it will:

1. Try-but-fail to add a user record
2. Not send an invite email
3. Update the preorder_information status to "School will be contacted shortly"

So we're then left with a situation requiring manual intervention from us. Do we know yet how are we going to detect these cases, and how we're going to resolve them?

Reply:
> So we're then left with a situation requiring manual intervention from us.

yep.

> Do we know yet how are we going to detect these cases, 

They will be the ones remaining in "school will be contacted" once every other school is on.

> and how we're going to resolve them?

I think we can onboard them once we support multi-org.